File: LUA/Cap08/c08ex03.lua
-- c08ex03.lua

print("CLASSIFICACAO DE NUMEROS (CRESCENTE)\n")

numeros = {}

for i = 1, 5 do
  io.write("Entre o " .. i .. "o. numero: ")
  numeros[i] = tonumber(io.read("*l"))
end

for i = 1, 4 do
  for j = i + 1, 5 do
    if (numeros[i] > numeros[j]) then
      local x = numeros[i]
      numeros[i] = numeros[j]
      numeros[j] = x
    end
  end
end

print()
for i = 1, 5 do
  print(i .. "o. numero: " .. numeros[i])
end

io.write("\nPressione <Enter> para encerrar... ")
io.read("*l")

File: PYTHON/c08ex03.py
# c08ex03.py

print("CLASSIFICACAO DE NUMEROS (CRESCENTE)\n")

numeros = {}

for i in range(1, 6):
  numeros[i] = int(input(f"Entre o {i}o. numero: "))

for i in range(1, 5):
  for j in range(i + 1, 6):
    if (numeros[i] > numeros[j]):
      x = numeros[i]
      numeros[i] = numeros[j]
      numeros[j] = x

print()
for i in range(1, 6):
  print(f"{i}o. numero: {numeros[i]}")

input("\nTecle <Enter> para encerrar... ")
